### DISCUSSION TITLE
Grapesjs can't be used in a React app ever since the last version update (0.17.28)

### DISCUSSION THREAD

**Discussion by ghost** [GrapesJS/grapesjs#3909] [Q&A]
Before the last version update, all I had to was use the `npm i grapesjs` command.
Then `import grapesjs from 'grapesjs';` and everything worked.

Now it just doesn't work. I don't know if it's me doing something wrong, or something changed.

The message error I receive is:
"Could not find a declaration file for module 'grapesjs'. 'c:/Users/XXX/test-grapesjs/node_modules/grapesjs/dist/grapes.min.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/grapesjs` if it exists or add a new declaration (.d.ts) file containing `declare module 'grapesjs';`"

It never happened before. What could it be?

**Comment by sandenson** [comment]
Are you using TypeScript or JavaScript in your project?

**Comment by artf** [comment]
The issue was fixed in https://github.com/artf/grapesjs/releases/tag/v0.17.29
